src/lib/support/logging/Constants.h

```diff
@@ -60,6 +60,7 @@ enum LogModule
     kLogModule_Automation,
     kLogModule_CASESessionManager,
     kLogModule_ICD,
+    kLogModule_FabricSync,
```

Comment:
both this and ICD seem to need adding to https://github.com/project-chip/connectedhomeip/blob/master/src/lib/support/logging/TextOnlyLogging.cpp#L152

Reply:
Acked